### ISSUE TITLE
Scott Melot Boss Art

### ISSUE THREAD

**Issue by goofanader** [goofanader/minna-no-fighting-game#2]
## Checklist
- [ ] Front Facing Sprite
- [ ] Idle Animation
- [ ] Point to the Right Sprite
- [ ] Attack 2 Animation
### Optional Animations
- [ ] Walking Animation
- [ ] Force Blast Sprite (hands and arms out to the sides of the screen, as if force pushing people away)
- [ ] Attack 1 Animation
## References
- **Boss Name:** Wizardo
- **References:** [Clothing Reference](http://innocent-w.jp/collaboration_e.html#c02), [Head Reference](https://media.licdn.com/mpr/mpr/shrink_100_100/AAEAAQAAAAAAAAOcAAAAJDAzNzhmNDc5LWZmMTgtNGRmMi1hZDJjLWI0NTUzZDk5ZmM2Ng.jpg), [Shining Wizard Attack](https://www.youtube.com/watch?v=HwsLMeQ6tNU)
- **Opening Monologue:** I can't....even....why am I here?
## Additional Items/Changes to the Reference
- **Hair:** Long hair (I did have that :P), you can color it however you want.
- **Face:** uuuuuh glasses?
- **Other:** hair piece rather than bonnet. You can recolor as you wish. The basic design is what I want since it would never fit me in reality :P
## Colors
- **Primary:** black
- **Secondary:** blue
- **Tertiary:** none
### Regular Attack: Knee 2 Face

It's a knee....to the face
### Special Attack: Shining Wizard

So basically it's a running kick but if you Google it you can actually see how it works?
## Last Remarks

I have many concerns, but not with this.....okay I do but...¯_(ツ)_/¯

**Comment by ghost** [comment]
Here's what I have of the sprite so far.
![image](https://cloud.githubusercontent.com/assets/15747311/11492787/93af48fe-97a6-11e5-9c5e-f17067d1d6d2.png)

**Comment by goofanader** [comment]
Scott sent feedback on the sprite:

"I didn't really understand the colors on the form, so here we go...

Primary - Burgundy (RGB 144, 0, 32)
Secondary - White
(I think he wanted the dress to be the colors that's in the reference picture -Phyllis)

Hair...not going to lie my actual hair looks weird so may I have a different hairstyle? Here is a reference, the color it is currently is fine, I'd just like a different style. Or not, yeah <a href="http://www.dolluxe.com/collections/classicwavy/products/classic-wavy-black-lolita" target="_blank">this</a> is better :<

Let me know if you need anything else."

**Comment by goofanader** [comment]
Emily is in charge of Scott now~

**Comment by goofanader** [comment]
![scott](https://cloud.githubusercontent.com/assets/799112/12842173/d0e212e6-cba5-11e5-89b4-f79376fa27be.png)

Scott, is this more what you meant?
